### ISSUE TITLE
Client behavior with multiple write regions

### ISSUE THREAD

**Issue by skurik** [Azure/azure-cosmos-dotnet-v3#1899]
I would like to clarify something I believe is not fully addressed in the [documentation](https://docs.microsoft.com/en-us/azure/cosmos-db/tutorial-global-distribution-sql-api?tabs=dotnetv2%2Capi-async#preferred-locations).

We are starting to use multiple write regions in our Cosmos account. Under normal circumstances, our application runs in West Europe, but should something happen there (as it did at the beginning of September), we have a North Europe region ready with the services duplicated (App Service, SQL, ...).

I basically want to know if it's safe _not_ to explicitly list the preferred regions when creating the client. The documentation says that

> If the __PreferredLocations__ property is not set, all requests will be served from the current write region.

So what happens if

1. The client is created without specifying __PreferredLocations__, the application starts.
2. The West Europe Cosmos region goes down.
3. Will the client start automatically issuing both writes and reads to the North Europe region?

In other words, does the _current write region_ change automatically?

Thank you!

**Comment by ealsur** [comment]
@skurik Does this documentation help? https://docs.microsoft.com/en-us/azure/cosmos-db/troubleshoot-sdk-availability

**Comment by skurik** [comment]
@ealsur Yes, didn't see that one before, thanks. You can close the issue.
